### PR TITLE
Added multiplier to locationLabel constraint

### PR DESCRIPTION
**Pull request by vm1000**
Regarding last issue(weather label was cut-off) I added multiplier to locationLabel. Not sure if it is good way to solve the issue, but now it seems working. 
Also added date in format that shown as example.
In example background is blue. Do I understand right that bg.color should change according to weather condition?

### CODE REVIEW

**Comment by ghost** [comment]
It was not intended for background color to change, but that is something you can add. How ever main functionality first, if you decide to add this feature, it will be last one

**Comment by ghost** [comment]
Did You pull latest changes into `develop` after merging PR, before making new branch `UI-implementation`.

Sourcetree

**Comment by ghost** [comment]
Multiplier, some voodoo, easier and more understandable ways to do it, add labels to `StackView` or add location label width constraint, or change weather label horizontal compression resistance priority

**Comment by vm1000** [comment]
> Did You pull latest changes into `develop`, before making new branch `UI-implementation`.
> 
> Sourcetree

Yes, I guess so. it wasn't necessary to do that?

File: simple-weather/simple-weather/Controller/ViewController.swift
Comment (ghost) [inline simple-weather/simple-weather/Controller/ViewController.swift]: Still, we are not displaying coordinates as location, we use coordinates to make API requests.

File: simple-weather/simple-weather/Model/CurrentWeather.swift
Comment (ghost) [inline simple-weather/simple-weather/Model/CurrentWeather.swift]: # **`!`**, we do not force unwrap
Either add default value `var cityName = ""` or have values as optionals `var cityName: String?`, optional for our case, to satisfy having uninitialised values in class.

File: simple-weather/simple-weather/Model/CurrentWeather.swift
Comment (ghost) [inline simple-weather/simple-weather/Model/CurrentWeather.swift]: Do not have any commented, or debug code (like setting label text as coordinates or printing coordinates) in Pull Request. Pull request must be completely clean, so it can be reviewed and instantly merged into develop.

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Again, this is `UI implementation` branch, why I see changes in files other than **.storyboard**?

**Comment by ghost** [comment]
> Yes, I guess so. it wasn't necessary to do that?

It was, but from Sourcetree history, does not look like that was done

**Comment by vm1000** [comment]
Thanks for review. I changed locationLabel width.
But all other actions that I do in code - how I could commit the UI changes only in that branch and changes in code in another. It means that I can work only in one feature at-time or it is possible somehow to split commits accordingly?

**Comment by vm1000** [comment]
> > Yes, I guess so. it wasn't necessary to do that?
> 
> It was, but from Sourcetree history, does not look like that was done

hmm, as I did - merge pull request -> confirm and changes are available in solution branch.

**Comment by vm1000** [comment]
> > > Yes, I guess so. it wasn't necessary to do that?
> > 
> > 
> > It was, but from Sourcetree history, does not look like that was done
> 
> hmm, as I did - merge pull request -> confirm and changes are available in solution branch.

or I needed to pull changes also locally in my source tree?

**Comment by ghost** [comment]
> Thanks for review. I changed locationLabel width.
> But all other actions that I do in code - how I could commit the UI changes only in that branch and changes in code in another. It means that I can work only in one feature at-time or it is possible somehow to split commits accordingly?

You can work on multiple features at a time. For example make UI in one branch, while that is reviewed to changes in Model in other branch. But not do both things in UI branch. You simply would be hopping from branch-to-branch all the time. Or you can wait for review to be completed, if other parts need those changes to continue, in such case ping me, that review is urgent, and I'll address it ASAP

**Comment by vm1000** [comment]
Screenshot 2019-04-24 at 11 11 24


Now that I changed width I have yellow error, that tells about label expected width and actual width mismatch, so where is that expected width. Where I can change that?

**Comment by ghost** [comment]
> > > > Yes, I guess so. it wasn't necessary to do that?
> > > 
> > > 
> > > It was, but from Sourcetree history, does not look like that was done
> > 
> > 
> > hmm, as I did - merge pull request -> confirm and changes are available in solution branch.
> 
> or I needed to pull changes also locally in my source tree?

As with previous PR.

- Create `new` branch and checkout `new` branch from `develop`
- Implement solution, and when all is complete create PR `develop` <- `new`
- PR is **reviewed**, add changes if necessary
- PR is **approved** marge PR (now in server changes from `new` are inside `develop`)
- Delete `new` branch
- Checkout `develop` (you'll that it is one-behind `develop` from server, since locally `new` change are not in `develop`)
- Pull  remote `develop` to local `develop` (so remote `develop` with changes from `new` now are in local `develop`)
- Now you are back as square 1. You have only `develop` branch. With latest changes locally as well as remote. Repeat.

**Comment by ghost** [comment]
> Screenshot 2019-04-24 at 11 11 24
> 
> Now that I changed width I have yellow error, that tells about label expected width and actual width mismatch, so where is that expected width. Where I can change that?

Select label to which pending changes must be applied
Add changes
Or if yellow warning pop-up that you have shown, simply **update frames**

**Comment by vm1000** [comment]
Screenshot 2019-04-24 at 12 10 48


Seems that I accidentally made new branch (Added date) that overtakes develop branch. Is it possible to make step back?
PS. before I made new branch the existing code in my project somehow deleted (I mean the changes that I made in UI implement branch) and I wasn't able to add those changes to UI-implement branch locally, only develop branch was active.

**Pull request merged**
